src/Paket.Core/ScriptGeneration.fs

```diff
@@ -121,10 +123,11 @@ module ScriptGeneration =
       input.OrderedRelativeDllReferences
       |> Seq.map (sprintf """#r "%s" """)
 
-    depLines
-    |> fun lines -> Seq.append lines framworkRefLines
-    |> fun lines -> Seq.append lines dllLines
-    |> fun lines -> Seq.append lines [ sprintf "System.Console.WriteLine(\"Loaded {0}\", \"%s\");" packageName ]
+    let lines = Seq.concat [depLines; framworkRefLines; dllLines]
+
+    if Seq.isEmpty lines 
```

Comment:
Good point, although I'm wondering if a proper record or DU return type wouldn't be better.

```fsharp
type GeneratedScript =
| DoNotGenerate
| Generate of (lines: string list)
```

would provide a clearer message at call site and future script gen implementations.